### PR TITLE
Activate only legit images

### PR DESCRIPTION
**Pull request by flavio**
Ensure only the images shipped by RPMs that have not been tampered can be imported.

### CODE REVIEW

**Comment by flavio** [comment]
@mjura can you review it when you have some spare time?

File: walker_test.go [hunk 1]
```diff
@@ -9,19 +9,60 @@ import (
...
+		t.Errorf("walker error: %v", err)
+	}
+
+	if len(walker.Files) != 0 {
+		t.Error("Expected 0 matches")
+	}
+}
+
+func TestWalkerWithVerificationEnabled(t *testing.T) {
+	topDir := "/usr/lib64/coreutils"
```
Comment (mjura) [inline walker_test.go]: I would propose this simple change `topDir := "/usr/share/zoneinfo"`

File: walker_test.go [hunk 2]
```diff
@@ -9,19 +9,60 @@ import (
...
+
+	if _, err := os.Stat(topDir); os.IsNotExist(err) {
+		t.Skipf("skipped: %s does not exist", topDir)
 	}
 
+	walker := NewWalker(topDir, ".so")
```
Comment (mjura) [inline walker_test.go]: And here `walker := NewWalker(topDir, ".tab")`

**Comment by flavio** [comment]
@mjura thanks for the review, please ping @jordimassaguerpla to make sure we ship this change into the first CaaSP beta release

**Pull request merged**